components/bl-animatedBox-component/component.json

```diff
@@ -0,0 +1,197 @@
+      "showInSettings": true,
+      "defaultValue": "bounce",
+      "hasLogicHandler": false,
+      "handlerId": "typeLogic",
+      "options": [
+        {
+          "value": "bounce",
+          "label": "bounce"
+        },
+        {
+          "value": "bounceIn",
+          "label": "bounceIn"
+        },
+        {
+          "value": "bounceOut",
+          "label": "bounceOut"
+        },
+        {
+          "value": "flash",
+          "label": "flash"
+        },
+        {
+          "value": "flip",
+          "label": "flip"
+        },
+        {
+          "value": "flipInX",
+          "label": "flipInX"
+        },
+        {
+          "value": "flipInY",
+          "label": "flipInY"
+        },
+        {
+          "value": "heartBeat",
+          "label": "heartBeat"
+        },
+        {
+          "value": "hinge",
+          "label": "hinge"
+        },
+        {
+          "value": "jackInTheBox",
+          "label": "jackInTheBox"
+        },
+        {
+          "value": "jello",
+          "label": "jello"
+        },
+        {
+          "value": "pulse",
+          "label": "pulse"
+        },
+        {
+          "value": "rotateIn",
+          "label": "rotateIn"
+        },
+        {
+          "value": "rubberBand",
+          "label": "rubberBand"
+        },
+        {
+          "value": "shakeX",
+          "label": "shakeX"
+        },
+        {
+          "value": "shakeY",
+          "label": "shakeY"
+        },
+        {
+          "value": "swing",
+          "label": "swing"
+        },
+        {
+          "value": "tada",
+          "label": "tada"
+        },
+        {
+          "value": "wobble",
+          "label": "wobble"
+        },
+        {
+          "value": "zoomIn",
+          "label": "zoomIn"
+        },
+        {
+          "value": "zoomOut",
+          "label": "zoomOut"
+        }
+      ],
+      "settingTooltip": "Select the animation type of the component."
+    },
+    {
+      "type": "number",
+      "name": "duration",
+      "label": "Duration",
+      "settingTooltip": "Duration of the animation, in seconds.",
+      "showInSettings": true,
+      "defaultValue": "1",
+      "hasLogicHandler": false,
+      "handlerId": "durationLogic"
+    },
+    {
+      "type": "number",
+      "name": "delay",
+      "label": "Delay",
+      "showInSettings": true,
+      "defaultValue": "0",
+      "hasLogicHandler": false,
+      "handlerId": "delayLogic",
+      "settingTooltip": "Duration to delay the animation, in seconds."
+    },
+    {
+      "type": "number",
+      "name": "loop",
+      "label": "Loop",
+      "settingTooltip": "The number of times the animation should loop, 0 is equivalent to infinite.",
+      "showInSettings": true,
+      "defaultValue": "0",
+      "hasLogicHandler": false,
+      "handlerId": "loopLogic"
+    },
+    {
+      "type": "checkbox",
+      "name": "autoStart",
+      "label": "Start Play",
```

Comment:
Auto Play?
Instant Play?
Play on mount?
Play on appear?

Or something that says that animation will be played at component mount